README.md

```diff
@@ -56,4 +56,17 @@ Then visit http://localhost:8100
 
 > If you want to re-run the `--build` command, you may get an error about not having access to delete a `cache/` folder. Use `sudo` to delete this manually before retrying.
 
+### Static Docker build
 
+Additionally, the `Dockerfile-standalone` Dockefile allows to produce a lightweight static build which also does not need local files to run. This can be useful in certain scenarios, for example for sharing the build output easily.
```

Comment:
Nit: given that you are calling it "static" and not "standalone" in the documentation, shouldn't the Dockefile also be called static?

Reply:
The meaning of "does not need local files" is not clear to me, as it actually uses the local cache. Maybe it would be more accurate to say "does not need installing dependencies locally"? 

Second, I'm not sure I understand the use case this option is intended for.

Reply:
> Nit: given that you are calling it "static" and not "standalone"

Fair enough, if people find this interesting I'll rename it.

> it actually uses the local cache

It caches the output of the build step, but does not need to mount the local folder in runtime:

https://github.com/grafana/k6-docs/blob/aa4f410cf198cd13b283989798dd69091e4037fd/docker-compose.yml#L13